dev/design-studio/package.json

```diff
@@ -33,7 +33,7 @@
     "@sanity/color": "^2.1.20",
     "@sanity/icons": "^2.3.0",
     "@sanity/logos": "^2.0.2",
-    "@sanity/ui": "^1.3.3",
+    "@sanity/ui": "popover-fixes-canary",
```

Comment:
Leaving a small comment here to not forget to revert this version before merging 😄 
(I've totally never done it myself, nope)